src/jdk.javadoc/share/classes/jdk/javadoc/internal/html/HtmlTree.java

```diff
@@ -365,6 +375,44 @@ public static HtmlTree A(URI ref, Content body) {
                 .add(body);
     }
 
+    /**
+     * Creates an HTML {@code BODY} element with the given style.
+     *
+     * @param style the style
+     * @return the element
+     */
+    public static HtmlTree BODY(HtmlStyle style) {
+        return new HtmlTree(HtmlTag.BODY)
+                .setStyle(style);
+    }
+
+    private static HtmlTree BR_INSTANCE = unmodifiableTree(HtmlTag.BR);
```

Comment:
There is a failing test in `tools/javac/T8003967/DetectMutableStaticFields.java` because of this and the other new static fields in `HtmlTree.java` not being `final`. The fields should be declared as `final` anyway.

Reply:
Thanks; I guess that test is not in my standard set of tests for javadoc: mea culpa

Reply:
Ideally the test should not be buried in tools/javac. But that's a separate matter.